### PR TITLE
Add Integration component to IntegrationsPage

### PR DESCRIPTION
**Pull request by MohsinGhani**
https://github.com/kwickbit/kwickbit_frontend_v0/issues/12

### CODE REVIEW

**Comment by abstract829** [comment]
Please do not use too much hardcoded tailwind styles unless you need some specific color,

If you are taking the exact margins/paddings from the design, just use the closest one from tailwind

example: ```mt-[30px]``` -> ```mt-8 (32px)``` 
```pt-[8px]``` is the same as ```pt-2```
```text-[16px]``` is ```text-base```
etc...

please take a look at the styles and replace for the defaults tailwind offers

you can install [tailwind vscode extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) so when you writing code you can see what css do the classes apply

Captura de pantalla 2023-11-24 a la(s) 16 31 04

**Comment by MohsinGhani** [comment]
> Please do not use too much hardcoded tailwind styles unless you need some specific color,
> 
> If you are taking the exact margins/paddings from the design, just use the closest one from tailwind
> 
> example: `mt-[30px]` -> `mt-8 (32px)` `pt-[8px]` is the same as `pt-2` `text-[16px]` is `text-base` etc...
> 
> please take a look at the styles and replace for the defaults tailwind offers
> 
> you can install [tailwind vscode extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) so when you writing code you can see what css do the classes apply
> 
> Captura de pantalla 2023-11-24 a la(s) 16 31 04

Thank You @abstract829 for the review. I will be careful about it. Please feel free to add comments if you feel any kind of inconsistency in the code.

**Comment by MohsinGhani** [comment]
Hi @abstract829, can you please take a look at the code now?

Thank You!

File: src/components/integration/index.tsx
Comment (abstract829) [inline src/components/integration/index.tsx:1]:
```bash
<p className="text-base text-[#171A1FFF] font-bold">Quickbooks</p>
<p className="text-base text-[#9095A1FF] font-bold">Accounting</p>
```
Captura de pantalla 2023-11-27 a la(s) 08 28 25
Captura de pantalla 2023-11-27 a la(s) 08 28 31

besides these color you can see "neutral-500" and "neutral-900" which means you can use text-neutral-500 and text-neutral-900 in tailwind, please change

Captura de pantalla 2023-11-27 a la(s) 08 27 23

Here the button is cut off, you can remove the ```w-20``` and wrap the button in a div and should be fine, or make your own button is also fine for now, the primarybutton is not so dynamic

**Review by abstract829 — CHANGES_REQUESTED** [review]
Great, looks a lot better, I leave some small improvements and we are ok

**Review by abstract829 — APPROVED** [review]

**Pull request merged**